### PR TITLE
fix: Allows snapping to grid when using multi arrow and line

### LINKED ISSUE #2009: Multipoint lines/arrows are not snapping to grid

### PR DESCRIPTION
**Pull request by mauriceLC92**
This PR fixes #2009 and allows for the snapping to the grid when using a multi arrow or multi-line element

### CODE REVIEW

File: src/components/App.tsx [hunk 1]
```diff
@@ -2075,9 +2075,19 @@ class App extends React.Component<ExcalidrawProps, AppState> {
             document.documentElement.style.cursor = CURSOR_TYPE.POINTER;
           }
           // update last uncommitted point
+          const lastUncommittedPoint = () => {
```
Comment (dai-shi) [inline src/components/App.tsx:2078]: I prefer making it a ternary instead of a function, but it might be just a preference.

**Comment by dwelle** [comment]
Ideally, the point preview (while moving mouse) should snap to grid as well (same as it does when creating a new point via Alt/Option while editing an existing line):

excal_create_multipoint_snap_to_grid

File: src/components/App.tsx (hunk 1 above)
Comment (mauriceLC92) [reply src/components/App.tsx:2078]: Happy to change to ternary if needed...I personally preferred the function though:)

**Comment by mauriceLC92** [comment]
Will update to include point preview

**Comment by lipis** [comment]
A friendly reminder @mauriceLC92 :)

**Comment by mauriceLC92** [comment]
@lipis reviving this now! 🙏

File: src/components/App.tsx (hunk 1 above)
Comment (mauriceLC92) [reply src/components/App.tsx]: Ternary added

**Comment by dwelle** [comment]
@mauriceLC92 forgot about this PR. What's the current status? AFAIK this PR doesn't do anything other than aligning to grid points in one branch which are already grid-aligned :).

**Pull request closed without merging**